lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -6,7 +6,7 @@
 from django.db import migrations, models
 from opaque_keys.edx.django.models import CourseKeyField, UsageKeyField
 
-import coursewarehistoryextended.fields
+from courseware.fields import UnsignedBigIntAutoField
 
 
 class Migration(migrations.Migration):
```

Comment:
There are additional references to `coursewarehistoryextended` in this file.  Please update all

Reply:
Ah right.  Thanks!